apps/chat/src/utils/app/data/application-service.ts

```diff
@@ -0,0 +1,123 @@
+import { Observable, map } from 'rxjs';
+
+import {
+  ApplicationDetailsResponse,
+  ApplicationListItemModel,
+  ApplicationListResponseModel,
+  ApplicationMoveModel,
+  CreateApplicationModel,
+} from '@/src/types/applications';
+import { EntityType } from '@/src/types/common';
+import { HTTPMethod } from '@/src/types/http';
+import { DialAIEntityModel } from '@/src/types/models';
+
+import { ApiUtils } from '../../server/api';
+import { constructPath } from '../file';
+import { BucketService } from './bucket-service';
+
+export class ApplicationService {
+  public static create(
+    applicationName: string,
+    applicationData: CreateApplicationModel,
+  ): Observable<ApplicationListItemModel> {
+    const bucket = BucketService.getBucket();
+    return ApiUtils.request(
+      constructPath(
+        'api',
+        'applications',
+        bucket,
+        ApiUtils.encodeApiUrl(applicationName),
+      ),
+      {
+        method: 'POST',
```

Comment:
```suggestion
        method: HTTPMethod.POST,
```
pay attention on [this comment](https://github.com/epam/ai-dial-chat/pull/1851/files#r1711340231)